cmd.go

```diff
@@ -110,35 +109,37 @@ func Run(request transform.PluginRequest) (transform.PluginResponse, error) {
 		return transform.PluginResponse{}, err
 	}
 
-	switch u.GetKind() {
-	case "Build":
-		logger.Info("found build, adding to whiteout")
+	if authorizationGroup == u.GetObjectKind().GroupVersionKind().GroupKind().Group {
 		whiteOut = true
-	case "BuildConfig":
-		logger.Info("found build config, processing")
-		patch, err = UpdateBuildConfig(u, inputFields)
-	case "Pod":
-		logger.Info("found pod, processing update default pull secret")
-		patch, err = UpdateDefaultPullSecrets(u, inputFields)
-	case "Route":
-		logger.Info("found route, processing")
-		patch, err = UpdateRoute(u)
-	case "ServiceAccount":
-		if inputFields.StripDefaultRBAC && (u.GetName() == "builder" || u.GetName() == "deployer") {
+	} else {
```

Comment:
Instead of doing this if/else...I think it would be easier to read and reason about if we went to:

```
if authorizationGroup == ... {
  	return transform.PluginResponse{
		Version:    string(transform.V1),
		IsWhiteOut: true,
		Patches:    patch,
	}, nil
}
```

then allow the `switch u.GetKind() {` to live peacefully :sunglasses: